vendors/carbonblack.json

```diff
@@ -1,8 +1,19 @@
 {
   "name": "Carbon Black",
```

Comment:
Where would we represent VMWare?

Reply:
The `name` should arguably be changed to "VMWare Carbon Black". We also represent this in the `aliases` property.